docs/contracts/uniswapx/guides/priorityorderreactor.md

```diff
@@ -11,12 +11,49 @@ Alice submits a PriorityOrder offering 1 ETH in exchange for a minimum of 1000 U
 
 If we assume a filler has a desired margin of 10% of the total profit, the best price is 1090 USDC. This would be 900 bps of improvement. This filler would convert bps to mps (see below for details) to get 900 * 1000 = 900,000 mps of improvement. Thus they would set priorityFee of 900,000 wei on their fill transaction. Keep in mind that this is additional to the base fee.
 
-# Details for Fillers
-- The interface is exactly the same as the current production version of UniswapX, but there will be a new reactor address which will require new approvals.
+# Important considerations
+- The PriorityOrderReactor is only meant to be used on chains which order transactions by priority fee. 
 - We do not plan to run any preliminary auctions for the start price of these orders, rather we set a minimum price that each order must be executed at.
-- Each order is only executable after a certain block specified by the user. You can expect this block to be a few blocks in the future from when you receive the order. To ensure the best UX for our users, Uniswap Labs will have the ability to make the start block earlier by cosigning the order. We expect nearly all orders to be filled in the first possible block.
-- For every wei of priority fee above a certain threshold (an optional order specified value), the user is owed 1 milli-bps more of their output token. Milli-bps (or MPS) are one-thousandth of a basis point.
-- PriorityOrders are only supported on chains which order transactions by priority fee — currently limited to OP stack L2s. Because of this ordering policy, the winning fill transaction necessarily must have also given the user the best execution.
+- Each order is only executable after a certain block specified by the user. This block will be a few blocks in the future from when the order is made available through the UniswapX orders endpoint. To ensure the best UX for our users, Uniswap Labs has the ability to make the start block earlier by cosigning the order.
+- Only the fill transaction with the highest priority fee will win the order, all other transactions will revert on chain.
+- To minimize the gas used on reverting transactions, we revert early if the order is already filled or is not fillable yet.
+- For every wei of priority fee above a certain threshold (an optional value specified in the order), the user is owed 1 milli-bps more of their output token (or less of their input token). 
+- Milli-bps (or MPS) are one-thousandth of a basis point.
+
+## Retrieving and Executing Signed Orders
+All signed Priority Orders created through the Uniswap UI will be available via the UniswapX Orders Endpoint. We have [swagger documentation](https://api.uniswap.org/v2/uniswapx/docs) but see below for a quick example curl.
+
+```
+GET https://api.uniswap.org/v2/orders?orderStatus=open&chainId=8453&orderType=Priority
```

Comment:
Is this for the new version v2 
don't we need this `&orderType=Dutch_V2` in the call?

Reply:
nope this is for a new experimental order type PriorityOrders which are returned by `Priority`